modules/@apostrophecms/modal/ui/apos/components/AposModalConfirm.vue

```diff
@@ -120,9 +120,9 @@ export default {
   padding: 60px;
 }
 
-.apos-confirm__icon {
-  width: 60px;
+.apos-confirm__logo {
   height: 60px;
+  fill: var(--a-black);
```

Comment:
This feels... bad since it's assigning a fill meant for a specific SVG to any logo that's passed in on the `img` tag. `fill` won't have any impact there, so I guess it's okay.